systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/ServiceUtils.java

```diff
@@ -79,11 +80,24 @@ public static void waitForNodePortService(String serviceName) throws Interrupted
     public static void waitForServiceDeletion(String serviceName) {
         LOGGER.info("Waiting for Service {} deletion in namespace {}", serviceName, kubeClient().getNamespace());
 
+        kubeClient().deleteService(serviceName);
```

Comment:
This should be just `wait` for the `service` deletion. That means, you need to delete it in the test and then use the `waitForServiceDeletion`.